edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/deployment/KubernetesDeploymentMapper.cs

```diff
@@ -41,6 +42,9 @@ public class KubernetesDeploymentMapper : IKubernetesDeploymentMapper
             string proxyTrustBundlePath,
             string proxyTrustBundleVolumeName,
             string proxyTrustBundleConfigMapName,
+            string persistentVolumeName,
```

Comment:
I was thinking they might get used, but they're not at this time, so I should remove them until they need to be sent.

Reply:
I do use pvname and storage class, but it's only to check if they are set.  If one or the other is set, that indicates that we use a PVC for volumes rather than a storage class.  I changed it to pass in a boolean, but I might want to keep the logic here.  Opinions?

Reply:
I chose to pass down the VolumeName and StorageClass, Set them as options, and made the decision in a new method.